src/ralph/cli.py

```diff
@@ -40,7 +37,6 @@
 
 
 @click.group(name="ralph")
-@click_log.simple_verbosity_option(get_root_logger())
 def cli():
     """Ralph is a stream-based tool to play with your logs"""
 
```

Comment:
I was wondering why passing verbosity options to ralph stopped having any effect on the logging.

Here `click_log.simple_verbosity_option` sets up the default logging configuration according to the verbosity options passed in the command line.

Later (line 43) `configure_logging` overwrites all the logging configuration according to environment variables / configuration file.